_layouts/500.html

```diff
@@ -0,0 +1,48 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    500 internal server error
+{%- endblock %}
+
+{% block desc -%}
+
+{%- endblock %}
+
+{% block content_modifiers -%}
+	error_main
+{%- endblock %}
+
+{% block content_main %}
+    <div class="error_copy_container">
+        <section class='error_heading u-mt30'>
+            <h1 class='superheader'>500 internal server error</h1>
+            <p class='short_desc'>We can’t find the page you were
+                looking for, but here are
+                a number of other ways you
+                might try to get there.
+            </p>
+        </section>
+        <aside class='block padded-container u-mt30 u-mb60 '>
+            <div class="content-l content-l__large-gutters padded-container_body">
+                <div class="content-l_col content-l_col-1-2 ">
+                    <p>
+                        Explore our other useful tools and resources
```

Comment:
Haha, they're pretty thick :eyeglasses:. I'm ineligible for Lasik due to my prescription being too strong :sob: